spec/forms/itt_start_date_form_spec.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe IttStartDateForm, type: :model do
```

Comment:
Reckon this might need one of these https://github.com/DFE-Digital/register-trainee-teachers/blob/b26ff6d77431ffa245b8d67e8ef3ae7a08da698b/spec/forms/withdrawal_form_spec.rb#L48

Reply:
Irrelevant as `itt_start_date ` is synonymous with  `course_start_date`.

Reply:
Then don't we need to validate it the same way we do for the `course_start_date`? eg:

<img width="878" alt="Screenshot 2021-07-27 at 10 56 41" src="https://user-images.githubusercontent.com/616080/127135248-a2096d8d-1b74-44df-b819-1e88bae04026.png">

Currently the `itt_start_date` allows a crazy old date:

![date](https://user-images.githubusercontent.com/616080/127135352-d1567828-4262-4134-93dd-06d16770b826.gif)

Not sure if you want to address it in this PR as it's getting quite big but maybe a follow up ticket?

Reply:
carded as https://trello.com/c/fmTreVUE/2364-itt-start-date-validation